src/common/header/HeaderNav.jsx

```diff
@@ -67,6 +67,18 @@ const HeaderNav = ({ showBrowse }) => {
               <span className="sr-only">Close</span>
             </button>
           </li>
+          {showToggleMenu && (
```

Comment:
@yeshwantthota We should have the activities on condition also, right? Otherwise the toggle bar browse is going to appear in the default mode also.